changelog/fragments/02-document-k8s-1-30-changes.yaml

```diff
@@ -0,0 +1,352 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >      
+      For Go-based, Helm-based and Ansible-based operators this release moves to Kubernetes 1.30 API's and Kubebuilder
+      v4 Scaffolding, specifically utilizing the v4.1.1 version. The update to Kubebuiler results in some scaffolding
+      changes which more information can be found below:
+      - Discontinue usage of [kube-rbac-proxy](https://github.com/brancz/kube-rbac-proxy) in the default
+      scaffolding of new projects. For further information,
+      see: [Action Required: Ensure that you no longer use gcr.io/kubebuilder images](https://github.com/kubernetes-sigs/kubebuilder/discussions/3907)
+      - The `go/v2` or `go/v3` layouts have been removed, you must upgrade to `go/v4` to be compatible with this release and future updates.
+      To know how to upgrade,check the [migration documentation](https://book.kubebuilder.io/migration/v3vsv4).
+      - Re-introduces authn/authz protection for the metrics endpoint using [`WithAuthenticationAndAuthorization`](https://github.com/kubernetes-sigs/controller-runtime/blob/v0.18.4/pkg/metrics/filters/filters.go#L35) 
+      provided by controller-runtime. which usage was [discontinued in the project](https://github.com/kubernetes-sigs/kubebuilder/discussions/3907). 
```

Comment:
```suggestion
      provided by controller-runtime instead of kube-rbac-proxy;which usage was [discontinued in the project](https://github.com/kubernetes-sigs/kubebuilder/discussions/3907). 
```

Reply:
Maybe?